recipes/r-pythoninr/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = 'PythonInR' %}
+{% set version = '0.1-4' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/{{ name }}_{{ version }}.tar.gz
+  sha256: ee9d5c33580b9273d80e43ed4d83dae81de2fc35cc92e310feefe5e4c12c394f
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
```

Comment:
You need to add Python as both a build and run requirements as you had originally done. From the [CRAN](https://cran.r-project.org/package=PythonInR) page:

> SystemRequirements: Python (>= 2.7.0) with header files and shared library (UNIX) / static library (Windows)

Adding `  - python` should do the trick, and then the CI services will build the package for all conda-forge supported Python versions ([source](https://github.com/conda-forge/staged-recipes/blob/092427d211f0e1e4eba17025fd06c65057f5271d/recipes/example/meta.yaml#L30)).